### ISSUE TITLE
Make Playbook available for openSUSE systems

### ISSUE THREAD

**Issue by ggeorgg** [ReinerNippes/nextcloud13#124]
I would like to adapt the playbook so that it can also setup nextcloud on openSUSE systems. I would like to work on this task and I would love to see my changes merged into your repository. Do you think this is possible?

**Comment by ReinerNippes** [comment]
Of course it's possible. 

First: It should happen in the "nextcloud-reloaded" branch. Please don't create merge request against master.

You may use https://github.com/ReinerNippes/ansible_var_dump to check all ansible facts. (pckage_mgt, major_version etc.pp.)

In each task/main.yml you find a `include_tasks: '{{ ansible_pkg_mgr }}.yml'` statemant. That leads normally to a yum.yml, apt.yml, dnf.yml. I guess you have to create a file "zypper.yml" to put all installation tasks inside. 

In the vars/main.yml you find variables that define os  specific list of packages, file and/or folder names which also have to be extend with a "Suse" block. 

OK?

**Comment by ggeorgg** [comment]
Yes, that's clear. I have already done some work, but currently I am getting an "Internal Server Error" on my Suse Tumbleweed System... 
nginx, redis, mariadb, php-fpm are up and running. If I use "phpinfo" in the index.php file, I get all information about my system, but unfortunately somehow the original nextcloud index.php can not be loaded by php-fpm and I don't get any logs in the nginx error log file...

I think, I can try to create a pull request and we can discuss the changes. Some changes are really quite minor changes, but for other I had to change a bit more.

**Comment by ReinerNippes** [comment]
Did you create a fork?

**Comment by ggeorgg** [comment]
Yes, but I did not push anything yet. It is still all just local.

**Comment by ReinerNippes** [comment]
You could push your changes to a new branch and we could try to fix things in this branch.

**Comment by ggeorgg** [comment]
I have created a pull request. Let me know what you think of it!
